frontend/ui/screensaver.lua

```diff
@@ -385,7 +385,7 @@ function Screensaver:setup(event, fallback_message)
     if G_reader_settings:has(self.prefix .. "screensaver_type") then
         self.screensaver_type = G_reader_settings:readSetting(self.prefix .. "screensaver_type")
     else
-        if event then
+        if event and not G_reader_settings:readSetting("disable_fallback_msg") then
```

Comment:
For consistency's sake, inititialize it to `false` like the rest of the settings (c.f., top of this module), and switch to an `isTrue` check here ;).

Reply:
Done! Let me know if it looks good now.

Reply:
Yep, I just forgot to mention that I'd prefer is everything kept a `screensaver_` prefix, too ;).

Reply:
(And Luacheck's trailing whitespace warning @ L197 ;)).

Reply:
Sounds good. I renamed them accordingly.